### PR TITLE
jobs/sync-stream-metadata: fix --dryrun arg passing

### PR DESCRIPTION
**Pull request by jlebon**
We need the appended `--dryrun` to be on the same line as `aws s3 sync`.

### CODE REVIEW

File: jobs/sync-stream-metadata.Jenkinsfile [hunk 1]
```diff
@@ -17,8 +17,7 @@ cosaPod(configMaps: ["fedora-messaging-cfg"], secrets: ["fedora-messaging-coreos
         cmd = """
             aws s3 sync --acl public-read --cache-control 'max-age=60' \
                 --exclude '*' --include 'streams/*' --include 'updates/*' \
-                    ./ s3://fcos-builds
-        """
+                    ./ s3://fcos-builds"""
```
Comment (dustymabe) [inline jobs/sync-stream-metadata.Jenkinsfile:20]: I could see someone mistakenly putting back the `"""` on the next line. Any way we could trim the newline in the `def dry_run = shwrapCapture("${cmd} --dryrun").trim()` instead?

Comment (jlebon) [reply jobs/sync-stream-metadata.Jenkinsfile]: I changed this to just explicitly `trim()` the `cmd` after defining it.

**Review by dustymabe — APPROVED** [review]
LGTM. I agree it seems silly to `.trim()` right after you define the variable. A comment would help explain to people why we did that (optional).

**Comment by jlebon** [comment]
Added comment!

**Pull request merged**